.github/workflows/python-code-cov.yml

```
@@ -18,10 +21,10 @@ jobs:
       with:
         persist-credentials: false
 
-    - name: Set up Python 3.10
+    - name: Set up Python 3.11
       uses: actions/setup-python@v3
```

actionlint (1.7.9)

the runner of "actions/setup-python@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)